runtime/src/bank.rs

```diff
@@ -1144,6 +1144,11 @@ pub struct NewBankOptions {
     pub vote_only_bank: bool,
 }
 
+#[derive(Debug, Default)]
+pub struct BankTestConfig {
+    pub secondary_indexes: Option<AccountSecondaryIndexes>,
```

Comment:
I don't think this needs to be an Option, does it?
As with the data passed through to accounts_db, I think this can just be `AccountSecondaryIndexes`, which has everything off by default.

Reply:
I was thinking int he future we may want to add more options to the struct and you may not want to provide every option in that case. This making everything Option<> in the struct forces the code to account for it to not be there. Technically I can remove the Option<> now, but then code that uses it will not be conditional (be it simpler), but in the future if you make it optional then you have to go back and add in the conditional logic.

Reply:
changed